tests/report.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import easypost
+from constants import API_KEY as api_key
+
+easypost.api_key = api_key
+
+class ReportTests(unittest.TestCase):
+
+  def test_report(self):
+    report = easypost.Report.create(
+        start_date="2012-12-01",
+        end_date="2013-01-01",
+        type="shipment"
+    )
+
```

Comment:
Since the classing is currently not working as intended, I'd consider adding something like `assert report.__class__ == easypost.Report` after the create and retrieve steps to confirm that it is properly casting to a Report